backend_jobs/ingestion_pipeline/unit_tests/firestore_database_test.py

```diff
@@ -0,0 +1,108 @@
+"""
+  Copyright 2020 Google LLC
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+"""
+ 
+import unittest
+import datetime
+from backend_jobs.ingestion_pipeline.pipeline_lib import firestore_database
+from backend_jobs.pipeline_utils import data_types
+
+
+class TestFirestoreDatabase(unittest.TestCase):
+    """ This class tests helper functions from the firebase_database class.
+    """
+  
+    def test_get_geo_hashes_map1(self):
+        """ This test, given values of latitude and longitude,
+        expects to receive a map object with all the geohashes.
```

Comment:
the docstring doesn't match the test, where you test one item in the dict, not all of them

Reply:
fix docstring